content/applications/finance/fiscal_localizations/indonesia.rst

```diff
@@ -181,3 +201,31 @@ QR-code` option to `QRIS`.
 
 Ensure that the :guilabel:`Recipient Bank` is the one you configured, as Odoo uses this field to
 generate the QRIS QR code.
+
+POS payment method configurations
+---------------------------------
+
+If you installed Point Of Sale, you can also add a new payment method for PoS via QRIS.
+Go to :menuselection:`Point of Sale --> Configuration --> Payment Methods`, create a new payment
+method.
+
+.. image:: indonesia/indonesia-qris-payment-method.png
+    :alt: QRIS Payment Method configuration
+
+Afterwards, add this payment method the PoS configurations that allow the payment.
+
+.. image:: indonesia/indonesia-qris-pos-config.png
+    :alt: PoS store configuration
```

Comment:
```suggestion
.. note:: If a session is currently open for the selected point of sale, it will have to be closed
   to add a new payment method.
```
This screenshot does not add much value and we're no longer using any markups in our screenshots. I think this note offers more help.